src/style.css

```diff
@@ -1,3 +1,66 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use the `*` selector as it impacts performance. Set styles only for elements that require them. Consider setting your base styles on the `html` and `body` elements or using class selectors instead.